examples/service-catalog/service-catalog-rbac.yaml

```diff
@@ -26,8 +26,9 @@ objects:
   roleRef:
     kind: ClusterRole
     name: servicecatalog-serviceclass-viewer
-  groupNames:
-  - system:authenticated
+  subjects:
+  - kind: Group
```

Comment:
s/Group/SystemGroup/

Reply:
> s/Group/SystemGroup/

That concept disappeared when we collapsed onto upstream.

Reply:
Interesting - I didn't see `SystemGroup` as a supported value from the API server when I goofed it at first.